src/GNDStk/Node/src/call.hpp

```diff
@@ -12,9 +12,9 @@
 // Those encode (via their type) whether we're accessing metadata or children.
 
 // Note: for variant-based meta_t and child_t keywords, you'll still need to
-// call meta<RESULT>() or child<RESULT> directly. Implementing the variant
-// versions here would bulk up this file, and you'd need to write the operator
-// call in functional form, e.g. mynode.operator()<RESULT>(...), to use it.
+// call meta<TYPE>() or child<TYPE> directly. Implementing the variant versions
```

Comment:
I'm not sure `TYPE` is that much better than `RESULT` here. It's not a reason to reject this Pull Request, but I'm wondering why the change.

Reply:
I originally used RESULT to mean, "result of the query." (As in, the type of the result, given the type system I've put together for this.)

But the system works not just for queries [or existing GNDS hierarachies], but for building brand-new hierarachies. That is to say, you can both read and write.

Given that it works both ways, I decided that the general and agnostic term TYPE was better than the term RESULT.